modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2407,10 +2421,18 @@ private DynamicObjectDefinitionTable _getDynamicObjectDefinitionTable(
 		ObjectDefinition objectDefinition =
 			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 
-		return new DynamicObjectDefinitionTable(
-			objectDefinition,
+		List<ObjectField> objectFieldList =
```

Comment:
As we discussed, it makes more sense to restrict the fields when creating the [selectExpressions](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L3052), instead of when creating the dynamic table, since it should accurately represent the database table.